liquibase-integration-tests/src/test/groovy/liquibase/command/ChangelogSyncIntegrationTest.groovy

```diff
@@ -0,0 +1,36 @@
+package liquibase.command
+
+import liquibase.Scope
+import liquibase.command.core.ChangelogSyncCommandStep
+import liquibase.command.core.helpers.DatabaseChangelogCommandStep
+import liquibase.command.core.helpers.DbUrlConnectionCommandStep
+import liquibase.command.util.CommandUtil
+import liquibase.extension.testing.testsystem.DatabaseTestSystem
+import liquibase.extension.testing.testsystem.TestSystemFactory
+import liquibase.extension.testing.testsystem.spock.LiquibaseIntegrationTest
+import spock.lang.Shared
+import spock.lang.Specification
+
+@LiquibaseIntegrationTest
+class ChangelogSyncIntegrationTest extends Specification {
+    @Shared
+    private DatabaseTestSystem h2 = Scope.currentScope.getSingleton(TestSystemFactory).getTestSystem("h2") as DatabaseTestSystem
+
+    def "run changelogSync from CommandStep"() {
```

Comment:
Very minor thing, but I would change the test name to something like "Validate deployment id is generated for a changelogSync execution" or something like that.

Reply:
Good call! Updated the test name!